_api-reference/snapshots/create-snapshot.md

```diff
@@ -73,7 +72,7 @@ Upon success, the response content depends on whether you include the `wait_for_
   "accepted": true
 }
 ```
-To verify that the snapshot was created, use the [Get snapshot API]({{site.url}}{{site.baseurl}}/opensearch/rest-api/snapshots/get-snapshot), passing the snapshot name as the `snapshot` path parameter.
+To verify that the snapshot was created, use the [Get snapshot API]({{site.url}}{{site.baseurl}}/api-reference/snapshots/get-snapshot), passing the snapshot name as the `snapshot` path parameter.
```

Comment:
API isn't included in the linked heading so move it outside of the hyperlinked text.

`[Get snapshot]({{site.url}}{{site.baseurl}}/api-reference/snapshots/get-snapshot) API`